test/e2e/e2e.go

```diff
@@ -2406,59 +2420,61 @@ var _ = ginkgo.Describe("e2e delete databases", func() {
-				time.Sleep(cooldown)
-				continue
-			}
-			if err != nil {
-				return err
-			}
-			return nil
-		}
-		return fmt.Errorf("Gave up after waiting %v for pod %q to be %q: pod is not found", timeout, podName, desc)
-	}
-
-	// waitForPodToFinishFullRestart waits for a the pod to finish it's reset cycle and returns.
-	waitForPodToFinishFullRestart := func(f *framework.Framework, pod *v1.Pod) {
-		err := e2epod.WaitForPodCondition(f.ClientSet, pod.Namespace, pod.Name, "not ready", 5*time.Minute, testutils.PodNotReady)
-		if err != nil {
-			framework.Failf("pod %v is not arrived to not-ready state: %v", pod.Name, err)
-		}
-		// during this stage on the restarting process we can encounter "pod not found" errors.
-		// this type of errors is valid because the pod is during restart so it will have a period that it will not be available.
-		// so we will use "WaitForPodConditionAllowNotFoundErrors" in order to handle properly those errors.
-		err = WaitForPodConditionAllowNotFoundErrors(f, pod.Namespace, pod.Name, "running and ready", 5*time.Minute, testutils.PodRunningReady)
-		if err != nil {
-			framework.Failf("pod %v is not arrived to running and ready state: %v", pod.Name, err)
-		}
-		return
-	}
-
 	table.DescribeTable("recovering from deleting db files while maintain connectivity",
 		func(db_pod_num int, DBFileNamesToDelete []string) {
+			var (
+				db_pod_name = fmt.Sprintf("%s-%d", databasePodPrefix, db_pod_num)
+			)
 			if db_pod_num < haModeMinDb || db_pod_num > haModeMaxDb {
 				framework.Failf("invalid db_pod_num.")
 				return
 			}
 
+			// adding db file path
+			for i, file := range DBFileNamesToDelete {
+				DBFileNamesToDelete[i] = path.Join(dirDB, file)
+			}
+
 			framework.Logf("connectivity test before deleting db files")
 			framework.Logf("test simple connectivity from new pod to API server,before deleting db files")
 			singlePodConnectivityTest(f, "before-delete-db-files")
 			framework.Logf("setup two pods for continues connectivity test ")
 			syncChan, errChan := make(chan string), make(chan error)
 			go twoPodsContinuousConnectivityTest(f, ovnWorkerNode, ovnWorkerNode2, syncChan, errChan)
-			//wait for the connectivity test pods to be ready
+			// wait for the connectivity test pods to be ready
 			framework.Logf(<-syncChan + "delete and restart db pods.")
-			//start the db disruption - delete the db files and delete the db-pod in order to emulate the cluster/pod restart
-			deleteAndRestartDB(db_pod_num, DBFileNamesToDelete, containerToDeleteFrom)
-			time.Sleep(10 * time.Second)
+
+			// start the db disruption - delete the db files and delete the db-pod in order to emulate the cluster/pod restart
+
+			// Retrive the DB pod
+			dbPod, err := f.ClientSet.CoreV1().Pods(ovnNs).Get(context.Background(), db_pod_name, metav1.GetOptions{})
+			framework.ExpectNoError(err, fmt.Sprintf("unable to get pod: %s, err: %v", db_pod_name, err))
+
+			//check that all files exists on pod
```

Comment:
@OmerBenHayun could you put a space after the `//` to be consistent with other comments? Thanks!

Reply:
@dcbw  added spaces after the `\\` , could you please approve the Build CI to run?
Thanks.